### ISSUE TITLE
XR-transformer model hyperparameters setting

### ISSUE THREAD

**Issue by zqudm** [amzn/pecos#196]
Thank you for great work. I have several questions about the 
**Fast Multi-Resolution Transformer Fine-tuning for Extreme Multi-label Text Classification (https://arxiv.org/pdf/2110.00685.pdf)**
The first one is about the model hyperparameters setting. In the appendix section of  paper, Table 7 lists the model hyperparameters. In particular, $HLT_{prelim}$ and $HLT_{refine}$ define the structures of the preliminary and refined hierarchical label trees.  For example , for dataset Eurlex-4K,  $HLT_{prelim}$ is set to 16-256-3956,  what does it means?  and does 16 represent the cluster size at the first layer?  What is more, what is the relationship between the param **nr_splits** with 16-256-3956? Last but not least, why did you chose these settings? [https://github.com/amzn/pecos/tree/mainline/examples/xr-transformer-neurips21/params](url)  

Each instance in my dataset has 48 labels. how to set or fine tune the nr_splits and other important parameters such as $\alpha$,$\lambda$ etc.

**Comment by jiong-zhang** [comment]
Hi @zqudm , Table 7 in the paper list the number of nodes at each level of the HLT. For example `16-256-3956` means the HLT is 3 levels and has 16 nodes, 256 nodes and 3956 (leaf) nodes respectively. `nr_split` defines the branching factor of the tree, i.e. `16 * 16 = 256`. The leaf level cluster is controlled by the `max_leaf_size`, which is the maximum leaf nodes in a cluster. 

In your case there's no need to construct any tree as the number of labels is very small. You can `max_leaf_size` to be bigger than 48 to have a flattened OVA model.
